docs/howto/features/index.md

```diff
@@ -5,6 +5,57 @@ There are several extra features and services that can be enabled on the 2i2c Ju
 These add extra functionality, connect with cloud services, etc.
 See the sections below for more details.
 
+## Layers
+
+### Cloud
+1. Shared/Dedicated Cluster
+2. Dedicated nodepool on a shared cluster (recommended for events)
+
+### Hub User Access
+1. Authentication + authorization
+2. Anonymisation of users ids
+
+### Community customizations
+1. Community specific hub domain
+2. (mandatory) Hub login page template sections
+3. Self-configuration of hub login page from own GitHub repository
+4. Customizations of hub pages
+
+### Data
+1. Shared data directories
+2. Object storage buckets (persistent/scratch)
+  - Cloud Permissions
+    - (default) Buckets accessible from the Hub
+    - Buckets accessible from outside the Hub
```

Comment:
```suggestion
    - (default) Buckets accessible from the Hub's cloud account
    - Buckets accessible from other cloud accounts
```

Is this what we mean? "Outside the hub" doesn't really make sense to me

Reply:
Yesss! Thank you.